### DISCUSSION TITLE
When are we going to have a stable version

### DISCUSSION THREAD

**Discussion by ogheneovo12** [themesberg/flowbite-react#1522] [General]
One of the major issues, i had and have with this project, is the change/update of Major APIs, without backward compatibility, for every version in a space of 2 - 3 months, there are always major changes, the only product i have utilised this project in, i have decided to lock down on that version, due to the stress of migrations, and uncertainty of what will and when will it change.
I am looking for a much more stable alternative.

**Comment by SutuSebastian** [comment]
Hey there, stability and improved backwards compatibility is one of the hardest things to do in an open source project, but when everybody adds whatever piece or logic they think it's fine without considering the bigger picture or existing API's, it becomes impossible.

All of the breaking changes that were done up until today were crucial for prepping the base layer and general architecture of the library. And more will come.

Until we have the v1 production release, this project will only steer toward that, even if it will have breaking changes.

Codemods will come in the future as well for seamless migration process.

With that being said, trust me, i completely feel ur pain and frustration.

Better days will come.
